mikupad.html

```diff
@@ -2343,7 +2374,7 @@
 			window.removeEventListener('keydown', onKeyDown);
 			window.removeEventListener('keyup', onKeyUp)
 		};
-	}, [predict, cancel]);
+	}, [predict, cancel, undoAndPredict, undo, redo]);

```

Comment:
This change makes no sense afaict.
```suggestion
	}, [predict, cancel]);
```